app/scripts/modules/core/src/styleguide/public/styleguide.html

```diff
@@ -1154,11 +1154,11 @@ <h2 id="psg-link-11" class="psg-component-title">Layers</h2>
                   <p>Controlling the number of layers a page has with z-index selectors.
 Use the following selectors to get the specific layer. This eliminates the need to
 manually set z-index that can be difficult to track.</p>
-<pre><code><span class="hljs-number">1</span>. <span class="hljs-class">.layer-critical</span>    gives you <span class="hljs-tag">a</span> zindex of <span class="hljs-number">10000</span>
-<span class="hljs-number">2</span>. <span class="hljs-class">.layer-high</span>        gives you <span class="hljs-tag">a</span> zindex of <span class="hljs-number">9000</span>
-<span class="hljs-number">3</span>. <span class="hljs-class">.layer-medium</span>      gives you  <span class="hljs-tag">a</span> zindex of  <span class="hljs-number">8000</span>
-<span class="hljs-number">4</span>. <span class="hljs-class">.layer-low</span>         gives you <span class="hljs-tag">a</span> zindex of <span class="hljs-number">7000</span>
-<span class="hljs-number">5</span>. <span class="hljs-class">.layer-backdrop</span>    gives you <span class="hljs-tag">a</span> zindex of <span class="hljs-number">6000</span>
+<pre><code><span class="hljs-number">1</span>. <span class="hljs-class">.layer-critical</span> or <span class="hljs-function"><span class="hljs-title">var</span><span class="hljs-params">(--layer-critical)</span></span>    gives you <span class="hljs-tag">a</span> zindex of <span class="hljs-number">10000</span>
```

Comment:
did you add this manually or did the build auto-gen it?

Reply:
The build generated it. Want me to remove it? I have no idea what it is.

Reply:
it's a styleguide, rob

Reply:
I see that. I have no idea why my changes deltaed it.

Reply:
i don't have a particularly strong opinion, but it does clutter the diff 🤷‍♂️